src/Core/Domain/SqlManagement/CommandHandler/SaveSqlRequestSettingsHandler.php

```diff
@@ -57,6 +54,7 @@ public function __construct(ConfigurationInterface $configuration)
     public function handle(SaveSqlRequestSettingsCommand $command)
     {
         $this->configuration->set(SqlRequestSettings::FILE_ENCODING, $this->getEncodingFileValue($command));
+        $this->configuration->set(SqlRequestSettings::FILE_SEPARATOR, $this->getFileSeparatorValue($command));
```

Comment:
```suggestion
        $this->configuration->set(SqlRequestSettings::FILE_SEPARATOR, $command->getFileSeparator());
```
This would have been enough, no need to copy the exact same thing as the encoding value. For encoding it makes more sense because the function actually transforms the value.